dvc/state.py

```diff
@@ -469,7 +469,9 @@ def remove_unused_links(self, used):
                 continue
 
             actual_inode = get_inode(path)
-            actual_mtime, _ = get_mtime_and_size(path, self.repo.dvcignore)
+            actual_mtime, _ = get_mtime_and_size(
+                path, self.repo.tree.dvcignore
+            )
```

Comment:
Maybe we could pass it a tree instead? Seems to be natural. Maybe I'm missing something though.